scipy/sparse/csgraph/tests/test_graph_laplacian.py

```diff
@@ -57,13 +57,15 @@ def test_laplacian_value_error():
 
 
 def test_symmetric_graph_laplacian():
-    symmetric_mats = ('np.arange(10) * np.arange(10)[:, np.newaxis]',
-            'np.ones((7, 7))',
-            'np.eye(19)',
-            'sparse.diags([1, 1], [-1, 1], shape=(4,4))',
-            'sparse.diags([1, 1], [-1, 1], shape=(4,4)).todense()',
-            'np.asarray(sparse.diags([1, 1], [-1, 1], shape=(4,4)).todense())',
-            'np.vander(np.arange(4)) + np.vander(np.arange(4)).T')
+    symmetric_mats = (
+        'np.arange(10) * np.arange(10)[:, np.newaxis]',
+        'np.ones((7, 7))',
+        'np.eye(19)',
+        'sparse.diags([1, 1], [-1, 1], shape=(4, 4))',
+        'sparse.diags([1, 1], [-1, 1], shape=(4, 4)).toarray()',
+        'np.asarray(sparse.diags([1, 1], [-1, 1], shape=(4, 4)).toarray())',
```

Comment:
this `np.asarray()` call is redundant now

Reply:
This change was actually against the original intention, which checked Laplacian with np.matrix input. I have fixed this now.